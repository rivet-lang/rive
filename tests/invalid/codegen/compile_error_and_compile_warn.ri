fn main() {
    compile_warn!("a simple warn");
    compile_error!("a simple error");
}
