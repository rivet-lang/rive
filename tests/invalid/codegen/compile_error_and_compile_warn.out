tests/failing/codegen/compile_error_and_compile_warn.ri:2:5: warning: a simple warn
    |
  2 |     compile_warn!("a simple warn");
    |     ^
tests/failing/codegen/compile_error_and_compile_warn.ri:3:5: error: a simple error
    |
  3 |     compile_error!("a simple error");
    |     ^
rivetc: error: could not compile package `core`, aborting due to previous error; 1 warning emitted